src/sass/main.scss

```diff
@@ -0,0 +1,24 @@
+//TODO: temporaty styles to see if gulp task is working
+$font-stack: Helvetica, sans-serif;
+$primary-color: #333;
+
+body {
+	font: 100% $font-stack;
+	color: $primary-color;
+
+	a {
+		text-decoration: none;
+	}
+	.container {
+		margin: 30px 0;
```

Comment:
cool, now I can see comments, do not forget to write comments in "Files changed" tab